nodes/utilities/ScMathsOp.py

```diff
@@ -5,13 +5,36 @@
 from bpy.types import Node
 from .._base.node_base import ScNode
 
+op_items = [
+    # (identifier, name, description)
+    ("ADD", "X + Y", "Addition"),
+    ("SUB", "X - Y", "Subtraction"),
+    ("MULT", "X * Y", "Multiplication"),
+    ("DIV", "X / Y", "Division"),
+    None,
```

Comment:
Thanks for this..
I didn't know you could use "None" to create a separator in the menu xD